Modules/Test/classes/class.ilObjTestGUI.php

```diff
@@ -1348,25 +1348,6 @@ public function importVerifiedFileObject()
             $map = $imp->getMapping();
             $map->addMapping('Modules/Test', 'tst', 'new_id', $newObj->getId());
             $imp->importObject($newObj, $fullPath, $fileName, 'tst', 'Modules/Test', true);
-        } else {
-            $qtiParser = new ilQTIParser(ilSession::get("tst_import_qti_file"), ilQTIParser::IL_MO_PARSE_QTI, $questionParentObjId, $_POST["ident"] ?? '');
-            if (!file_exists(ilSession::get("tst_import_results_file"))
-                && (!isset($_POST["ident"]) || !is_array($_POST["ident"]) || !count($_POST["ident"]))) {
-                $qtiParser->setIgnoreItemsEnabled(true);
-            }
-            $qtiParser->setTestObject($newObj);
-            $qtiParser->startParsing();
```

Comment:
FYI @mjansenDatabay, @mbecker-databay, @dsstrassner, @kergomard, @thojou 
Neue Erkenntnis, das ganze ist sogar noch simpler als gedacht. Das ist bereits Code der aus ilTestImporter kopiert wurde und diese wichtige Zeile fehlt. Damit funktioniert das ganze auch wieder und würde den PR zu einem Einzeiler machen
Ich finde trotzdem den Ansatz besser, die Manifest Datei mit in die Zip zu packen und sich so auch die Duplikation von Code zu sparen.

```php
$newObj` = $qtiParser->getTestObject();
````